hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/workflows/QueryContext.java

```diff
@@ -35,6 +37,14 @@ public interface QueryContext {
     @NonNull
     Query query();
 
+    /**
```

Comment:
Title of the PR is a bit misleading given the very first thing here is an enhancement to the QueryContext.  Which I understand from services modularization meeting is - if not a big deal, at least a deal.  Perhaps it should be mentioned in the PR/commit message?

Reply:
True, added to the PR text.